### ISSUE TITLE
Does this project support EAP ?

### ISSUE THREAD

**Issue by zzycn003** [layeh/radius#100]
This project is great. but does this support EAP ? 

I asked because I didn't find any EAP-Message stuff in codes

**Comment by maxzerker** [comment]
no, it doesn't, I've just finished EAP-PEAP support but it's not ready for production yet.

**Comment by zzycn003** [comment]
> no, it doesn't, I've just finished EAP-PEAP support but it's not ready for production yet.

good to know, thanks. did you submit your changes ?

**Comment by sliusardanya** [comment]
> no, it doesn't, I've just finished EAP-PEAP support but it's not ready for production yet.

@maxzerker 
is there any news?

**Comment by jamiesun** [comment]
@danyanya   I have integrated freeradius to solve this problem  https://github.com/talkincode/toughradius/tree/main/freeradius

**Comment by BeryJu** [comment]
I've implemented EAP with support for EAP-TLS, EAP-GTC and EAP-MSCHAPv2 as a standalone library: https://github.com/BeryJu/radius-eap

(There's also a writeup about it here in case you're curious https://beryju.io/blog/2025-05-implementing-eap/)
